acceptance/tests/task_local.rb

```diff
@@ -38,9 +38,6 @@
       #!/bin/sh
       echo "$PT_greetings from $(whoami)"
       FILE
-      # TODO: Use input_method: both (default) once bug BOLT-1283 is fixed
```

Comment:
The bug described in https://tickets.puppetlabs.com/browse/BOLT-1283 is solved by dropping the wrapper script entirely.